tests/includes/getBlogPostsTest.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+require '../../includes/getBlogPosts.php';
+
+class getBlogPostTest extends PHPUnit_Framework_TestCase{
+    public function testGetBlogListinggood(){
```

Comment:
add line break above first function to make reading easier
word good in function name needs a capital G
move { to new line